dipy/segment/select.py

```diff
@@ -0,0 +1,174 @@
+"""
+Select streamlines
+"""
+
+
+from warnings import warn
+import numpy as np
+from nibabel.affines import apply_affine
+from dipy.tracking.utils import streamline_near_roi
+from dipy.core.geometry import dist_to_corner
+
+
+def reduce_rois(rois, include):
+    """
```

Comment:
Moved the function into tracking.streamline and eliminated the unnecessary white-space.